From 2e1607448d3b2dc5b4005429878beabe087c449d Mon Sep 17 00:00:00 2001
From: Jason Liu <jason.hui.liu@nxp.com>
Date: Wed, 25 Oct 2017 06:29:19 +0800
Subject: [PATCH 2689/5242] MLK-16688 driver: media: mxc_jpeg: Remove the
 unused function to kill the build warnnings
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit  4b4014be763fcd0280730bce643c5d43466a2952 from
https://source.codeaurora.org/external/imx/linux-imx.git

The patch fixes the following build warnnings by removing unused function:

drivers/media/platform/imx8/mxc-jpeg.c:228:13: warning: ‘print_output’ defined
but not used [-Wunused-function]
 static void print_output(void *addr)
             ^~~~~~~~~~~~

This patch also does the minor clean up by removing some commented-out code

Signed-off-by: Jason Liu <jason.hui.liu@nxp.com>
Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/imx8/mxc-jpeg.c |   14 --------------
 1 file changed, 14 deletions(-)

diff --git a/drivers/media/platform/imx8/mxc-jpeg.c b/drivers/media/platform/imx8/mxc-jpeg.c
index e3a0df6..c013e2e 100644
--- a/drivers/media/platform/imx8/mxc-jpeg.c
+++ b/drivers/media/platform/imx8/mxc-jpeg.c
@@ -225,17 +225,6 @@ static int enum_fmt(struct mxc_jpeg_fmt *mxc_formats, int n,
 	return 0;
 }
 
-static void print_output(void *addr)
-{
-	int i;
-
-	for (i = 0x0; i < 0x200; i += 4) {
-		printk(KERN_DEBUG "%02x %02x %02x %02x\n", ((char *)addr)[i],
-		       ((char *)addr)[i+1], ((char *)addr)[i+2],
-		       ((char *)addr)[i+3]);
-	}
-}
-
 void mxc_jpeg_addrs(struct mxc_jpeg_desc *desc, struct vb2_buffer *b_base0_buf,
 struct vb2_buffer *bufbase_buf, int offset)
 {
@@ -358,9 +347,6 @@ static void mxc_jpeg_device_run(void *priv)
 		mxc_jpeg_set_desc(dma_handle, jpeg->base_reg, slot);
 		mxc_jpeg_go(jpeg->base_reg);
 	}
-	//print_descriptor_info(desc);
-	////print_output(testaddro);
-	//print_cast_decoder_info(jpeg->base_reg);
 	spin_unlock_irqrestore(&ctx->mxc_jpeg->hw_lock, flags);
 }
 static int mxc_jpeg_job_ready(void *priv)
-- 
1.7.9.5

